amd_openvx_extensions/amd_rpp/include/vx_ext_rpp_version.h

```diff
@@ -38,8 +38,8 @@ SOFTWARE.
 extern "C" {
 #endif
 #define VX_EXT_RPP_VERSION_MAJOR 3
-#define VX_EXT_RPP_VERSION_MINOR 0
-#define VX_EXT_RPP_VERSION_PATCH 1
+#define VX_EXT_RPP_VERSION_MINOR 1
```

Comment:
@kiritigowda Please confirm if this version change is ok since it is not changing major version

Reply:
VX_RPP Version update LGTM